openslides_backend/action/actions/user/create.py

```diff
@@ -38,6 +39,7 @@ class UserCreate(
             "organisation_management_level",
             "is_present_in_meeting_ids",
             "committee_ids",
+            "committee_$_management_level",
```

Comment:
Yes, I think the latter possibility is easier.

Reply:
What's the problem about the implementation? If the user has CML permission for C1, he can manage a meeting within C1, whether the user is member of C1 or not! May be he's the manager of C1, without being member of C1 as Ortsverband, but member of Bundesverband C2.
If you don't agree:
Solution 2 is easier, but hard to understand for the user! Therefore I would prefer number 1, setting automatically the membership, when he gots the CML permission. To remove it has to be done on both manually.

Reply:
Ah ok, that's how you interpreted it, but not how it was intended: Analogous to a meeting, where you can only have permissions (by being part of a group) if you are part of the meeting, one can only have a CML in a committee if he is part of that committee. I guess this wasn't clear enough from the wiki...

Reply:
Actually I didn't thought about that, but as I said: it is inconsistent if you can manage a committee where you are not a member. The actual user of the UI won't ever see this since he cannot access meeting where he is not assigned, so this case does not happen in the UI, but it leaves a security issue open: When you remove a user from a committee, he currently still can manage it.

Reply:
Today I talked with Emanuel and we will stay with committee manage permission implying being part of a committee. 
I think then we should do it analogous to meeting: remove the field committee_ids (or use it as calculated field like meeting_ids) and use the committee_$_management_level analogue to group_$_ids to prove being part of committee. For not beeing manager of a committee we would use something like "can_see". 
IMO this would be consistent, simple to check (see PR with checking necessary dependencies) and easier to understand.  
Perhaps we could talk about on Monday.